tests/torch/data/reference_graphs/pattern_merging/basic_pattern.dot

```diff
@@ -1,12 +1,12 @@
 strict digraph  {
-"0 /A_0" [associated_ip_node_keys="{'POST HOOK 0 /A_0'}", node_type="InsertionPointGraphNodeType.OPERATOR", regular_node_data="0 /A_0"];
-"4 /B_0" [associated_ip_node_keys="{'PRE HOOK 0 4 /B_0', 'POST HOOK 4 /B_0'}", node_type="InsertionPointGraphNodeType.OPERATOR", regular_node_data="4 /B_0"];
+"0 /A_0" [associated_ip_node_keys="{'POST HOOK 0 /A_0'}", is_merged=False, node_type="InsertionPointGraphNodeType.OPERATOR", regular_node_data="0 /A_0 A"];
+"4 /B_0" [associated_ip_node_keys="{'POST HOOK 4 /B_0', 'PRE HOOK 0 4 /B_0'}", is_merged=False, node_type="InsertionPointGraphNodeType.OPERATOR", regular_node_data="4 /B_0 B"];
 "POST HOOK 0 /A_0" [insertion_point_data="TargetType.OPERATOR_POST_HOOK /A_0", node_type="InsertionPointGraphNodeType.INSERTION_POINT"];
 "PRE HOOK 0 1 /conv2d_0" [insertion_point_data="TargetType.OPERATOR_PRE_HOOK 0 /conv2d_0", node_type="InsertionPointGraphNodeType.INSERTION_POINT"];
 "POST HOOK 3 /RELU_0" [insertion_point_data="TargetType.OPERATOR_POST_HOOK /RELU_0", node_type="InsertionPointGraphNodeType.INSERTION_POINT"];
 "PRE HOOK 0 4 /B_0" [insertion_point_data="TargetType.OPERATOR_PRE_HOOK 0 /B_0", node_type="InsertionPointGraphNodeType.INSERTION_POINT"];
 "POST HOOK 4 /B_0" [insertion_point_data="TargetType.OPERATOR_POST_HOOK /B_0", node_type="InsertionPointGraphNodeType.INSERTION_POINT"];
-"1 /conv2d_0\n2 /batch_norm_0\n3 /RELU_0\n" [associated_ip_node_keys="{'POST HOOK 3 /RELU_0', 'PRE HOOK 0 1 /conv2d_0'}", node_type="InsertionPointGraphNodeType.OPERATOR", regular_node_data="1 /conv2d_0"];
+"1 /conv2d_0\n2 /batch_norm_0\n3 /RELU_0\n" [associated_ip_node_keys="{'PRE HOOK 0 1 /conv2d_0', 'POST HOOK 3 /RELU_0'}", is_merged="[<nncf.common.graph.graph.NNCFNode object at 0x7f4ff5a2ed90>, <nncf.common.graph.graph.NNCFNode object at 0x7f4ff5a2e290>, <nncf.common.graph.graph.NNCFNode object at 0x7f4ff5a2e790>]", node_type="InsertionPointGraphNodeType.OPERATOR", regular_node_data="1 /conv2d_0 conv2d"];
```

Comment:
is it expected content of `is_merged` node?

Reply:
Not really, will fix

Reply:
Done